doc/sphinx/source/utils/RTW/add_a_recipe.rst

```diff
@@ -26,48 +26,80 @@ How to add a recipe to the |RTW|
-   ``esmvaltool/utils/recipe_test_workflow/flow.cylc`` file.
+.. _run_a_recipe_in_the_rtw:
 
-   .. hint::
-      If the recipe takes less than 10 minutes to run then it should be added
-      to the ``fast`` option. Recipes that take longer than ten minutes should
-      be added to the ``medium`` option.
+Run a Recipe in the |RTW|
+-------------------------
 
-   .. hint::
-      The line added should follow the format of ``recipe_new_recipe, \``,
-      unless the line is the last one in the list, in which case the line added
-      should follow the format of ``recipe_new_recipe``.
-
-#. If the duration of the recipe is larger than the value specified by the
-   ``execution time limit`` option in the ``[[COMPUTE]]`` section in the
-   aforementioned site-specific ``.cylc`` file, and / or the memory usage of
-   the recipe is larger than the value specified by the ``--mem`` option in the
-   ``[[[directives]]]`` section in the ``[[COMPUTE]]`` section, add a section
-   (in alphabetical order) to this file as shown below (round the duration to
-   the nearest second)::
-
-     [[process<fast=recipe_albedolandcover>]]
-     # Actual: 0m31s, 2.5 GB on 2024-04-08.
-     execution time limit = PT2M
-     [[[directives]]]
-         --mem = 3G
+#. The recipe should now be added to your ``<site>-recipes.cylc`` file. (Where
+   ``<site>`` is your site). Find this in the
+   ``esmvaltool/utils/recipe_test_workflow/site/`` directory. Add the recipe in
+   alphabetical order for easy user reference:
+   :ref:`Currently tested recipes <currently_tested_recipes>`.
+
+#. ``<site>-recipes.cylc`` contains **two lists of dictionaries**. The lists
+   are ``FAST_RECIPES`` and ``MEDIUM_RECIPES``.
 
    .. hint::
-      The ``fast`` key in the example task definition above
-      (``[[process<fast=recipe_albedolandcover>]]``) should match name of the
-      option the recipe was added to in the ``[task parameters]`` section in
-      the ``esmvaltool/utils/recipe_test_workflow/flow.cylc`` file
+      ``FAST_RECIPES`` take *less* than 10 minutes to run at your site.
+      ``MEDIUM_RECIPES`` take *more* than 10 minutes.
+
+#. Add the recipe to one of lists as as a dictionary of ``key, value`` pairs.
+   E.g.::
+
+      {
+         'recipe_path': 'recipe_a_fast_recipe',
+         'actual': '1m30s, 1.5 GB on 2025-01-01',
+         'max_time': 'PT2M',
+         'max_memory': '2G',
+      }
+
+   Add the following information for each key:
+
+   .. list-table::
+      :widths: 25 75
+      :header-rows: 1
+
+      * - Key
+        - Value
+      * - ``recipe_path``
+        - The path to the recipe. Recipe paths are specified relative to
+          ``esmvaltool/recipes``. For recipes in subdirectories, ``--`` stands
+          for ``/`` since the latter is an illegal char.
+      * - ``actual``
+        - A note of  the recipe's actual resource usage. From the successful run
+          of the recipe on the compute server at your site.
```

Comment:
Would it be worth including the text in point 3 above here (i.e. how to get this information from the ESMValTool log)?

Reply:
@chrisbillowsMO and I worked together on the documentation and we moved these parts of the documentation together, so hopefully it's clear where the information comes from: a353bf7 🎉